src/components/common/CreateColonyWizard/CreateColonyWizard.ts

```diff
@@ -19,9 +20,10 @@ const stepArray: StepType[] = [
 ];
 
 export type FormValues = {
-  tokenName: string;
-  tokenSymbol: string;
+  tokenName?: string;
+  tokenSymbol?: string;
   tokenAddress: string;
+  token?: Token | null;
```

Comment:
Either `tokenName`/`tokenSymbol` (when creating a new token) or `token` will be defined

Reply:
Storing `token` in hook-form state allows us to simplify the validation (we used to run the get token query twice, this allows us to simply check if token is null or not)